src/Servers/Kestrel/samples/HttpClientApp/Program.cs

```diff
@@ -3,17 +3,27 @@
 
 using System.Net;
 using System.Net.Http;
+using System.Security.Cryptography.X509Certificates;
+using Microsoft.AspNetCore.Testing;
+
+// Console.WriteLine("Ready");
+// Console.ReadKey();
```

Comment:
Meant to still be here?

Reply:
Yeah, these are useful if you try to debug the Http3SampleApp. You either have to start the client first and then debug the server, or start both with debugging enabled on the server, and this pause lets you wait until the server is ready.